packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1346,6 +1347,7 @@
   "Username_wants_to_start_otr_Do_you_want_to_accept": "__username__ wants to start OTR. Do you want to accept?",
   "Users": "Users",
   "Users_in_role": "Users in role",
+  "Users_online": "__total__ users online",
```

Comment:
I'd change this key to `__total__users_online` instead of `Users_online`. Just because we might want to translate "Users online" at some point in the future, and this would make the key ambiguous.